templates/pyramid/files/app.py

```diff
@@ -0,0 +1,16 @@
+import os
```

Comment:
The `services.yaml` file includes both MySQL and Redis, but neither are being used, or even have demo code commented out.  Is there anything we can provide in that regard?  (I don't know Python/Pyramid well enough to say what that would even look like.)